### PR TITLE
mirror:Add the printing of event time

### PR DESCRIPTION
**Pull request by dormanze**
## Community Contribution License
All community contributions in this pull request are licensed to the project maintainers
under the terms of the [Apache 2 license](https://www.apache.org/licenses/LICENSE-2.0). 
By creating this pull request I represent that I have the right to license the 
contributions to the project maintainers under the Apache 2 license.

## Description
The event time is printed in watch mode.

## How to test this PR?
setp1:
keep watch
```
mc mirror -w test/bucket1 test/bucket2 --quiet
```
setp2:
upload or remove file.
image

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Optimization (provides speedup with no functional changes)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [ ] Fixes a regression (If yes, please add `commit-id` or `PR #` here)
- [ ] Unit tests added/updated
- [ ] Internal documentation updated
- [ ] Create a documentation update request [here](https://github.com/minio/docs/issues/new?label=doc-change,title=Doc+Updated+Needed+For+PR+github.com%2fminio%2fmc%2fpull%2fNNNNN)

### CODE REVIEW

File: cmd/mirror-main.go [hunk 1]
```diff
@@ -277,17 +279,31 @@ type mirrorJob struct {
...
 
 // String colorized mirror message
 func (m mirrorMessage) String() string {
-	return console.Colorize("Mirror", fmt.Sprintf("`%s` -> `%s`", m.Source, m.Target))
+	var msg string
+	if m.EventTime != "" {
+		msg = console.Colorize("Time", fmt.Sprintf("[%s] ", m.EventTime))
+	}
+	if m.EventType == notification.ObjectRemovedDelete {
+		return msg + "Removed " + console.Colorize("Removed", fmt.Sprintf("`%s`", m.Target))
```
Comment (shtripat) [inline cmd/mirror-main.go:299]: `Removed` twice?

File: cmd/mirror-main.go [hunk 2]
```diff
@@ -593,10 +619,6 @@ func (mj *mirrorJob) monitorMirrorStatus(cancel context.CancelFunc) (errDuringMi
 
 		if sURLs.SourceContent != nil {
 			mirrorTotalUploadedBytes.Add(float64(sURLs.SourceContent.Size))
-		} else if sURLs.TargetContent != nil {
```
Comment (shtripat) [inline cmd/mirror-main.go:596]: else part not needed?

**Comment by klauspost** [comment]
@dormanze Could you address the issues mentioned?

**Comment by dormanze** [comment]
> @dormanze Could you address the issues mentioned?

Okay, I've solved it.  Please review.  Thank you very much.

**Review by klauspost — APPROVED** [review]
LGTM

**Review by zveinn — APPROVED** [review]

**Review by donatello — APPROVED** [review]

**Pull request merged**